index.html

```diff
@@ -212,6 +212,33 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *@kudamndau* Contributor card END ________  -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">ldsuk00</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">@LdsUk00</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/your_user_handle" target="_blank">LdsUk00e</a>
+          </p>
+          <p class="about">I love the sense of contributing towards an open souce</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="Github">https://github.com/</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="CBTNUGGETS">https://www.cbtnuggets.com/</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Pluralsight’s">https://www.pluralsight.com/</a>
+              </li>
```

Comment:
Please be noted that the links must be placed inside the `href` attribute of `<a>` tag insted of `#` which is your case. 
To help you further understand, here is a sample code for the `resource` section: 
```
          <div class="resources">
            <p>3 Useful Dev Resources</p>
            <ul>
              <li>
                <a href="https://developers.cloudflare.com/" target="_blank" title="cloudflare-docs">Cloudflare Docs</a>
              </li>
              <li>
                <a href="https://nextjs.org/docs" target="_blank" title="nextjs-docs">Nextjs Docs</a>
              </li>
              <li>
                <a href="https://ui.shadcn.com/" target="_blank" title="shadcn-ui">Shadcn-ui</a>
              </li>
            </ul>
          </div>
```